django_prices_vatlayer/models.py

```diff
@@ -18,5 +18,11 @@ def __str__(self):
         return self.country_code
 
 
+class RateTypesManager(models.Manager):
```

Comment:
It would be better to use custom queryset and use `as_manager()` method in model (https://docs.djangoproject.com/en/1.10/topics/db/managers/#creating-a-manager-with-queryset-methods)